api/src/main/java/org/openmrs/module/patientflags/task/PatientFlagTask.java

```diff
@@ -72,12 +72,8 @@ public void generatePatientFlags(Flag flag) {
 	}
 
 	private static void generatePatientFlags(Flag flag, FlagService service) {
-
 		service.deletePatientFlagsForFlag(flag);
-		generatePatientFlagsForFlagAndPatient(flag,service);
-	}
 
-	private static void generatePatientFlagsForFlagAndPatient(Flag flag, FlagService service){
```

Comment:
why do these methods get merged?

Reply:
I created `generatePatientFalgsForFalgAndPatient(..)` method in[ this PR](https://github.com/openmrs/openmrs-module-patientflags/pull/91) and this used to generate patient flags without deleting the existing patient flag. With the new changes we can use `generatePatientFlags(Flag flag)` method.